.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "lmnopEnv\\Scripts\\python.exe"
+}
```

Comment:
This file causes conflicts and needs to be removed. Please remove this from your local git cache and push.
`git rm -r --cached .vscode`